### PR TITLE
Special case for page number 2

### PR DESCRIPTION
**Pull request by Flor-IT**
Hi,
in case of page number 2 it would be better to take the original url of category page instead of "?p=1". I´ve made it with a simple search/replace - could made with the ulr from controller.
Thanks
Oliver

### CODE REVIEW

**Comment by tim-bezhashvyly** [comment]
+1 on this. My SEO department also has also requested the same thing.

File: src/app/code/community/Dh/SeoPagination/Model/Paginator.php [hunk 1]
```diff
@@ -87,7 +87,12 @@ public function createLinks()
         
         //Determine exactly what needs to be output and 
         //add to the head block
-        if (!$pager->isFirstPage() && !$pager->isLastPage() && $numPages > 2 ) {
+        if ($pager->getCurrentPage() == 2) {
+            $moddedPreviousUrl =  str_replace('?p=1', '', $pager->getPreviousPageUrl());
```
Comment (tim-bezhashvyly) [inline src/app/code/community/Dh/SeoPagination/Model/Paginator.php]: What if URL contains other _GET parameters? E.g. layered navigation?

Comment (Flor-IT) [inline src/app/code/community/Dh/SeoPagination/Model/Paginator.php]: Good point. Should be considered. At the moment it´s not treated well

Comment (tim-bezhashvyly) [inline src/app/code/community/Dh/SeoPagination/Model/Paginator.php]: I will try to implement it.

**Comment by tim-bezhashvyly** [comment]
I guess this PR in not relevant anymore since #3 is now merged into master.

**Comment by drewdhunter-pers** [comment]
Closing as completed as per https://github.com/drewhunter/SeoPagination/pull/3

**Pull request closed without merging**